deimos/openssl/stack.d

```diff
@@ -89,7 +90,7 @@ char* M_sk_value()(_STACK* sk, size_t n) { return (sk ? sk.data[n] : null); }
 
 *******************************************************************************/
 
-version (DeimosOpenSSLv1_1)
+static if (OPENSSL_VERSION_AT_LEAST(1, 1))
```

Comment:
While it does no harm, there's perhaps no reason to have any backwards compatibility stuff in here (unless it's going to be backported to support openssl-1.0, but there's already a release for that).